src/data/balance/passives.ts

```diff
@@ -1,7 +1,15 @@
 import { Abilities } from "#app/enums/abilities";
 import { Species } from "#app/enums/species";
 
-export const starterPassiveAbilities = {
+export interface PassiveAbilities {
+  [key: integer]: Abilities
```

Comment:
```suggestion
  [key: number]: Abilities
```
Btw you should use `number` instead of `integer` (`integer` is just an alias for `number` and doesn't actually provide any additional type safety annoyingly).